packages/desktop/src/renderer/store/reducers.ts

```diff
@@ -57,7 +76,19 @@ const allReducers = combineReducers(reducers)
 
 export const rootReducer = (state, action) => {
   if (action.type === communities.actions.resetApp.type) {
-    state = undefined
+    const torBootstrapProcess = state.Connection.torBootstrapProcess
```

Comment:
I think it can possibly be misleading for someone in the future. I mean, this code doesn't look like it serves for resetting state anymore - for me it has something to do with monitoring tor connection. Excluding it into a separate function or at least adding some comment could be helpful maybe?